src/main/java/com/google/songgame/servlets/RoomServlet.java

```diff
@@ -53,6 +53,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     datastore.put(roomEntity);
   }
 
+  @Override
+  public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Map<String, String> roomProperties = readJSONFromRequest(request);
+
+    // Get room entity.
+    Query query = new Query("Room");
+    PreparedQuery results = datastore.prepare(query);
+    Entity roomEntity = results.asSingleEntity();
```

Comment:
maybe I'm not seeing it, but I'm not understanding how it is working with multiple rooms. I don't see querying by the rooms unique ID here.

Reply:
+1, I think you need to filter to a specific room here

Reply:
@alekseychaikovsky Yes Peter and I went back over it and I was actually only adding the user to the last created room. I just added a new query filter that finds the right room!